### ISSUE TITLE
How long is it supposed to install?

### ISSUE THREAD

**Issue by imrek** [saviorisdead/RustyCode#244]
After 2 hours of installation time, it still hasn't run though the entire process.

**Comment by W4RH4WK** [comment]
@imrek check out <https://marketplace.visualstudio.com/items?itemName=kalitaalexey.vscode-rust> as this repository is no longer maintained.

**Comment by imrek** [comment]
I guess a deprecation notice would be helpful in this case to prevent other people from trying to install it.

**Comment by W4RH4WK** [comment]
yes, sadly the author has gone inactive without any notification.
